packages/storybook/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 v0.40.0
 ------------------------------
+<<<<<<< HEAD
```

Comment:
Ups

Reply:
Cheers :D I need to revisit the changelogs shortly; got one more issue to resolve before its ready